Packs/CiscoFirepower/Integrations/CiscoFirepower/CiscoFirepower.py

```diff
@@ -1469,120 +3602,842 @@ def get_task_status_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict
+        'ciscofp-list-security-group-tags': list_security_group_tags_command,
+        'ciscofp-list-ise-security-group-tag': list_ise_security_group_tags_command,
+        'ciscofp-list-vlan-tags': list_vlan_tags_command,
+        'ciscofp-list-vlan-tags-group': list_vlan_tags_group_command,
+        'ciscofp-list-applications': list_applications_command,
+        'ciscofp-get-access-rules': get_access_rules_command,
+        'ciscofp-create-access-rules': create_access_rules_command,
+        'ciscofp-update-access-rules': update_access_rules_command,
+        'ciscofp-delete-access-rules': delete_access_rules_command,
+        'ciscofp-list-policy-assignments': list_policy_assignments_command,
+        'ciscofp-create-policy-assignments': create_policy_assignments_command,
+        'ciscofp-update-policy-assignments': update_policy_assignments_command,
+        'ciscofp-get-deployable-devices': get_deployable_devices_command,
+        'ciscofp-get-device-records': get_device_records_command,
+        'ciscofp-deploy-to-devices': deploy_to_devices_command,
+        'ciscofp-get-task-status': get_task_status_command,
+        'ciscofp-create-intrusion-policy': create_intrusion_policy_command,
+        'ciscofp-list-intrusion-policy': list_intrusion_policy_command,
+        'ciscofp-update-intrusion-policy': update_intrusion_policy_command,
+        'ciscofp-delete-intrusion-policy': delete_intrusion_policy_command,
+        'ciscofp-create-intrusion-rule': create_intrusion_rule_command,
+        'ciscofp-list-intrusion-rule': list_intrusion_rule_command,
+        'ciscofp-update-intrusion-rule': update_intrusion_rule_command,
+        'ciscofp-delete-intrusion-rule': delete_intrusion_rule_command,
+        'ciscofp-upload-intrusion-rule-file': upload_intrusion_rule_file_command,
+        'ciscofp-create-intrusion-rule-group': create_intrusion_rule_group_command,
+        'ciscofp-list-intrusion-rule-group': list_intrusion_rule_group_command,
+        'ciscofp-update-intrusion-rule-group': update_intrusion_rule_group_command,
+        'ciscofp-delete-intrusion-rule-group': delete_intrusion_rule_group_command,
+        'ciscofp-create-network-analysis-policy': create_network_analysis_policy_command,
+        'ciscofp-list-network-analysis-policy': list_network_analysis_policy_command,
+        'ciscofp-update-network-analysis-policy': update_network_analysis_policy_command,
+        'ciscofp-delete-network-analysis-policy': delete_network_analysis_policy_command,
+    }
 
-    LOG(f'Command being called is {command}')
+    demisto.debug(f'Command being called is {command}')
 
     try:
-        if demisto.command() == 'test-module':
-            return_outputs('ok')
+        client = Client(
+            base_url=base_url,
+            username=username,
+            password=password,
+            verify=verify_ssl,
+            proxy=proxy,
+        )
+
+        if command == 'test-module':
+            return_results('ok')
```

Comment:
not testing anything?

Reply:
This is from the old implementation.

This should be alright because if we've reached this point that means that the `__init__` which has a request within it (for the `bearer token`) has worked.

Reply:
please add this as a comment in 4428, so future devs know it :)

Reply:
Cool, please add a comment explaining it, and a unit test for failure as well.

Reply:
added a comment and a unit-test 🙂